File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long b = 0;
        int c = x;
        while (x>0){
            b = b*10 + x%10;
            x = x/10;
        }
        if (b == c){
            return true;
        }
        else {
            return false;
        }
    }
};

File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        b = 0
        c = x
        while (x>0):
            b = b*10 + x%10
            x = x//10
        
        if (b == c):
            return True
        else:
            return False
        